validation/helpers.py

```diff
@@ -1,7 +1,8 @@
 from os import fspath
 
 import divvunspell
-from difflib import Differ, SequenceMatcher, get_close_matches
+from difflib import Differ
+import hfst_optimized_lookup
```

Comment:
This should be added to the Pipfile.

Reply:
`difflib` seems to come with python, but I've added `hfst-optimized-lookup`